content/SCALE/SCALEUIReference/Apps/InstallCustomAppScreens.md

```diff
@@ -0,0 +1,274 @@
+| **Capabilities** | Click **Add** to display a **Add Capability** field. Click again to add another field. |
+| **Add Capability** | Enter a capability. |
+| **Configure Container User and Group ID** | Select to display the **Run Container as User** and **Run Container as Group** settings to add security context (`runAsUser` and `runAsGroup` variables). |
+| **Run Container As User** | Enter a numeric user ID for the container. Default is **568**. |
+| **Run Container as Group** | Enter a numeric group ID for the container. Default is **568**. |
+{{< /truetable >}}
+{{< /expand >}}
+
+### Scaling/Upgrade Strategy Settings
+**Scaling/Upgrade Strategy** settings configure how application upgrades replace pods.
+
+{{< trueimage src="/images/SCALE/23.10/InstallCustomAppScalingUpgradePolicy.png" alt="Scaling/Upgrade Strategy Settings" id="16: Scaling/Upgrade Strategy Settings" >}}
+
+Select **Create new pods and then kill the old ones** to recreate the container. 
+This retains the existing configuration and container until the upgrade completes before removing it.
+
+Select **Kill existing pods before creating new ones** to do rolling upgrades. 
+This removes the existing pod and start with a newly updated pod. 
+Killing existing pods is useful if your old pod is not functioning properly. 
+For fewer issues, select **Kill existing pods before creating new ones**.
+
+### Resource Reservation Settings
+**Resource Reservation** settings configure GPU devices detected by the system. Settings only display if the system detects the GPU device.
+
+Select the device on the **Select GPU** dropdown list of devices.
+Setting vary based on the device selected.
+
+### Resource Limits Settings
+
+**Resource Limits** settings specify the CPU and memory limits to place on the Kubernetes pod. 
+
+{{< trueimage src="/images/SCALE/23.10/InstallCustomAppAddResourceLimits.png" alt="Resource Limits Settings" id="17: Resource Limits Settings" >}}
+
+{{< expand "Settings Information" "v" >}}
+{{< truetable >}}
+| Setting | Description |
+|---------|-------------|
+| **Enable Pod resource limits** | Select to enable resource limits and display the **CPU Limit** and **Memory Limit** settings. |
+| **CPU Limit** | Enter the integer values with the suffix m (mill) you want to use to limit the CPU resource. For example, 1000m, 100, etc. |
+| **Memory Limit** | Enter the number of bytes you want to limit memory to. Follow the number with the quantity suffix, like E, P, T, G, M, k or Ei, Pi, Ti, Mi, Gi, Ki. For example, 129e6, 129m, 12897484800m, 123Mi, etc. |
+{{< /truetable >}}
+{{< /expand >}}
+
+### Portal Configuration Settings
+
+The **Portal Configuration** settings configure the web UI portal for the container.
+
+Select **Enable WebUI Portal (only supported in TrueNAS SCALE Bluefin)** to display the web portal configuration settings.
```

Comment:
It is what it says, but *why* does it say this in Cobia?

Reply:
I have no idea why devs kept this. I'll follow up on this one.